src/st/init_entity.h

```diff
@@ -7,8 +7,9 @@ void InitializeEntity(u16 arg0[]) {
     g_CurrentEntity->unk5A = *arg0++;
     g_CurrentEntity->palette = *arg0++;
 
-    enemyId = *arg0++;
-    g_CurrentEntity->enemyId = enemyId;
+    // n.b.! the post increment of arg0 is optimized out
+    // on the PS1 version, but not on the PSP version.
```

Comment:
Probably not necessary to have this comment? Either way is fine.

Reply:
PR checks will catch trying to remove the increment, but I made the comment to avoid future me trying to "optimize" by removing the increment and then failing to remember the PSP version needs it.

If it's too noisy, I'm fine getting rid of it.

Reply:
Nah, fair enough, go ahead and keep it commented if it's helpful